LICENSE.txt

```diff
@@ -0,0 +1,13 @@
+Copyright ArxanFintech Technology Ltd. 2018 All Rights Reserved.
```

Comment:
Please follow the project sdk-go-common, you are using a wrong license.txt file.

Reply:
done.